app/helpers/form_helper.rb

```diff
@@ -330,11 +330,11 @@ def check_required options, f, attr
     return ' *'.html_safe if required.nil? ? is_required?(f, attr) : required
   end
 
-  def blank_or_inherit_f(f, attr)
+  def blank_or_inherit_f(f, attr, blank_value = _("no value"))
```

Comment:
would it be better to use options hash? I'd like to avoid changing the method signature in future when we need one more optional argument.